### ISSUE TITLE
Give the game_name on the StreamCard a distinct color to be easily readable

### ISSUE THREAD

**Issue by 000744210** [Seldszar/Gumbo#41]
**Is your feature request related to a problem? Please describe.**
Problem: When reading over the list of streams it can be overwhelming to try to extract the game name from each card due to the stream name having the same text color

**Describe the solution you'd like**
Give the card a distinct color to be readable while skimming through the list.
I chose `text-red-400/50` as the color for the text in dark mode and it turned out nice.

**Describe alternatives you've considered**
Making the text bold would also give the same effect without adding a new color to the pallet.

**Additional context**
This is my solution to the problem using the text color described above.
![image](https://user-images.githubusercontent.com/63418620/165205601-45ec832b-b4f6-4ffa-8843-0f28938880c4.png)

**Comment by Seldszar** [comment]
![image](https://user-images.githubusercontent.com/3079214/170894237-0f12b85c-3057-44f8-9008-3fc1551d24fc.png)

What do you think of this option?

**Comment by Seldszar** [comment]
The previous change proposal will be available on the next version (1.11).

**Comment by placchio** [comment]
> ![image](https://user-images.githubusercontent.com/3079214/170894237-0f12b85c-3057-44f8-9008-3fc1551d24fc.png)
> 
> What do you think of this option?

I dont think this is good, it makes it really hard to read which should be always the number 1 priority i assume.
Can i change it back to what it was before?
